tests/bouncer/search_test.py

```diff
@@ -50,7 +50,8 @@ def test_configures_es_for_aws_auth(self, es_mock, awsauth_mock):
                                         use_ssl=True,
```

Comment:
I guess you fixed some PEP8 warnings in this file? Which is great, thanks! It's just that sometimes it makes a PR hard to review if there are unrelated changes to the problem you're trying to solve. Just a tip for the future :slightly_smiling_face: